### PR TITLE
Fix generating bc files for tests

### PR DESCRIPTION
**Pull request by lukekoko**
For [Test-Suite PR #39](https://github.com/SVF-tools/Test-Suite/pull/39), generating bc files need to be done before SVF build.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -120,6 +120,14 @@ fi
 export PATH=$LLVM_DIR/bin:$PATH
 echo "LLVM_DIR =" $LLVM_DIR
 
+# if test-suite is present then build bc files for testing
```
[the remark this answers is not in the document]
Comment (yuleisui) [reply build.sh:123]: Why does not generate the bc? It is a bit strange.

[the remark this answers is not in the document]
Comment (lukekoko) [reply build.sh:123]: It generates the bc but the tests are added before this. I'm not sure how to make it run generate_bc before the tests

**Comment by yuleisui** [comment]
CI is broken. Seems to be a permission problem.

https://travis-ci.org/github/SVF-tools/SVF/jobs/770702109#L4460

**Comment by yuleisui** [comment]
Can we disable testing on OSX since it can generate quite different bcs that break the CI?

**Pull request merged**